LottieUWP/LottieDrawable.cs

```diff
@@ -68,6 +68,43 @@ public LottieDrawable()
             Unloaded += UserControl_Unloaded;
         }
 
+        public CanvasCommandList GetCanvasImage(ICanvasResourceCreator resourceCreator, float scaleX, float scaleY)
```

Comment:
This method feels like a different necessity. I like it, but does not feels like it should be on the same PR.

Reply:
Sorry, I should delete this method before I create this PR.
I use it to render image, but it not work well on some times(Issues #37)